### PR TITLE
Add start dashboard proxy scripts

### PR DESCRIPTION
**Pull request by sam-roberts-bjss**
[AB#6578](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/6578)
[AB#6342](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/6342)

### CODE REVIEW

File: local-helm/start-dashboard-proxy.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+echo "Starting dashboard proxy"
+
+TOKEN=`kubectl -n kube-system describe secret default | grep token: | awk '{print $2}'`
+kubectl config set-credentials docker-desktop --token=$TOKEN
+
+DASHBOARDURL="http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/#!/overview?namespace=buyingcatalogue"
+echo "Copy"
```
Comment (KatieArmstr) [inline local-helm/start-dashboard-proxy.sh:9]: don't think you need this line as the link echoed below should be clickable?

**Comment by KatieArmstr** [comment]
could we combine this into launch-or-update-environment?

File: local-helm/start-dashboard-proxy.sh (hunk 1 above)
Comment (sam-roberts-bjss) [reply local-helm/start-dashboard-proxy.sh:9]: I think that depends on your terminal as mine wasn't clickable, but I can remove it if you like?

**Comment by sam-roberts-bjss** [comment]
The Kubernetes proxy command will block the terminal so I don't think we can combine it.

File: local-helm/start-dashboard-proxy.sh (hunk 1 above)
Comment (KatieArmstr) [reply local-helm/start-dashboard-proxy.sh:9]: mac 😄

Comment (KatieArmstr) [reply local-helm/start-dashboard-proxy.sh:9]: nah it is fine I just thought that was was bash did

**Comment by KatieArmstr** [comment]
> The Kubernetes proxy command will block the terminal so I don't think we can combine it.

I didn't think this would matter as if you update you can kill it and run the command? Just a thought as this is the way I run it so it would still mean 2 commands which is annoying

**Comment by sam-roberts-bjss** [comment]
> > The Kubernetes proxy command will block the terminal so I don't think we can combine it.
> 
> I didn't think this would matter as if you update you can kill it and run the command? Just a thought as this is the way I run it so it would still mean 2 commands which is annoying

I run the dashboard script once for the day & don't need to run it again as this is only required when you need to view the cluster information. I believe not everyone uses the same dashboard as @stuart-bass-bjss uses [Rancher](https://rancher.com/), which I still haven't managed to setup yet. But does look like a cleaner interface & doesn't require setting a config file every time to view the dashboard.

File: local-helm/start-dashboard-proxy.sh (hunk 1 above)
Comment (sam-roberts-bjss) [reply local-helm/start-dashboard-proxy.sh:9]: 👍

**Review by KatieArmstr — APPROVED** [review]

**Pull request merged**